pyccel/parser/semantic.py

```diff
@@ -1023,7 +1042,9 @@ def _visit_DottedVariable(self, expr, **settings):
 
         # look for a class method
         if isinstance(expr.rhs, Application):
-
+            methods = list(first.cls_base.methods) + list(first.cls_base.interfaces)
+            if any(isinstance(method, Interface) for method in methods):
+                errors.report('Generic methods are not supported yet', severity='fatal')
```

Comment:
Can we provide the method as a symbol?

Reply:
We should even have access to the line number in semantic with:
```
bounding_box=(self._current_fst_node.lineno, self._current_fst_node.col_offset)
```

Reply:
should I loop over the methods to get the one creating the problem, or the line number is enough?

Reply:
Yes, I think you should loop over the methods. We should not worry about performance here, because we are going to raise an error anyway!

Reply:
Is this conversation resolved? So no loop to get the method?

Reply:
I will add the loop

Reply:
I think it's good now